src/libs/Clipboard/index.ts

```diff
@@ -1,17 +1,40 @@
 // on Web/desktop this import will be replaced with `react-native-web`
-import {Clipboard} from 'react-native-web';
-import lodashGet from 'lodash/get';
+import {Clipboard as RNWClipboard} from 'react-native-web';
 import CONST from '../../CONST';
 import * as Browser from '../Browser';
+import {SetString, Clipboard} from './types';
 
-const canSetHtml = () => lodashGet(navigator, 'clipboard.write');
+type ComposerSelection = {
+    start: number;
+    end: number;
+    direction: 'forward' | 'backward' | 'none';
+};
+
+type AnchorSelection = {
+    anchorOffset: number;
+    focusOffset: number;
+    anchorNode: Node;
+    focusNode: Node;
+};
+
+type Nullable<T> = {[K in keyof T]: T[K] | null};
```

Comment:
Wondering if we have a dedicated space for such generic types 🤔 If not, let me know what you think about making a folder for such purpose? 👍

Reply:
Shared types can created as a file in `src/types/utils`, but the name of this one is little wrong for that it is doing.

Usually the `Nullable` type would be only `type Nullable<T> = T[K] | null;`, this one is iterating through the root keys of a object and setting them to nullable.